### DISCUSSION TITLE
In `format: pdf` include option to move `table of contents` to a new page

### DISCUSSION THREAD

**Discussion by filipemsc** [quarto-dev/quarto-cli#11740] [Feature Requests]
### Description

By default, when the TOC is enabled (toc: true), it is placed immediately after the title in the same page. I would like to request the addition of a native option to place the Table of Contents (TOC) on a separate page when generating PDF documents. This feature would enhance the flexibility of document formatting, particularly for academic and professional reports, where the TOC is typically expected to appear after the title page on a new page.

Thanks!

**Comment by mcanouil** [comment]
See

- https://github.com/quarto-dev/quarto-cli/discussions/11386
- https://quarto.org/docs/journals/templates.html#latex-partials

**Reply by Steinthal** [reply]
I'm using my own template based on the [Eisvogel pandoc template](https://github.com/Wandmalfarbe/pandoc-latex-template) and have changed the ```toc.tex``` partial to use an option ```toc-own-page```as follows (changes between ```$-- %```):

```text
$if(toc)$
$if(toc-title)$
\renewcommand*\contentsname{$toc-title$}
$endif$
$if(beamer)$
\begin{frame}[allowframebreaks]
$if(toc-title)$
  \frametitle{$toc-title$}
$endif$
  \setcounter{tocdepth}{$toc-depth$}
  \tableofcontents
\end{frame}

$-- % eisvogel: TOC on own page
$if(toc-own-page)$
\newpage
$endif$
$-- % 
$else$
{
$if(colorlinks)$
\hypersetup{linkcolor=$if(toccolor)$$toccolor$$else$$endif$}
$endif$
\setcounter{tocdepth}{$toc-depth$}
\tableofcontents

$-- % eisvogel: TOC on own page
$if(toc-own-page)$
\newpage
$endif$
$-- % 
}
$endif$
$endif$
$if(lof)$
\listoffigures
$endif$
$if(lot)$
\listoftables
$endif$
```
